### ISSUE TITLE
Can't look at Outfits: dresserID not set properly

### ISSUE THREAD

**Issue by emmamei** [emmamei/cdkey#609]
Dolly Minka said this:
```
[23:38:28] Minka (saraminka.nova): Are you looking at my outfits? it says [23:37] Dolly Minka's Key: You go to look in Dolly's closet for clothes, and find that Lady Mei メイ Mэй Moorcock-Stone is already there looking
```
Need to confirm that this "lock" is relinquished at the appropriate time.

**Comment by emmamei** [comment]
Dress.lsl seems to be missing a timer event? `dresserID` needs to be cleared after timer expires.

**Comment by emmamei** [comment]
We are attempting with `dresserID` to lock others out of dressing dolly when someone else is doing it. The way this is done has to be examined and tested.

**Comment by emmamei** [comment]
This problem was fixed by commit  f1af031. The variable `dresserID` was not getting cleared appropriately.
